### ISSUE TITLE
Automate Desktop files and folders cleanup!

### ISSUE THREAD

**Issue by Bhavya0304** [python-geeks/Automation-scripts#829]
## Managing Desktop directories and files

**Gets you the information on whats useful and whats not useful on desktop. So you can remove unwanted files and folders on desktop and also automate task of desktop cleanup** 

### Programming Language

- [x] Python

**Comment by pawangeek** [comment]
Hey, @Bhavya0304 So do we have to add the name or path of each unwanted file or what?

**Comment by Bhavya0304** [comment]
No, basically It will automatically get all the files from desktop recursively (files from folder included) and will show you all the files that are not being modified from a long time. So you can keep a track of unused files. Also the script provides you with a option to delete it.

**Comment by lolzone13** [comment]
Hey, could I work on this?

**Comment by WaniAthar** [comment]
@Bhavya0304 i can work on this

**Comment by Hasshey** [comment]
I would like to work on this, can this be assigned to me?

**Comment by Hasshey** [comment]
https://github.com/python-geeks/Automation-scripts/pull/1017 - raised a PR for this, I apologize for raising without the issue assigned to me - I waited for 24 hrs for anyone to respond and hence rasied a PR
